platform/o.n.bootstrap/launcher/windows/jvmlauncher.cpp

```diff
@@ -249,7 +249,7 @@ bool JvmLauncher::startInProcJvm(const char *mainClassName, const std::list<std:
             for (list<string>::const_iterator it = options.begin(); it != options.end(); ++it, ++i) {
                 const string &option = *it;
                 logMsg("\t%s", option.c_str());
-                if (option.find("-splash:") == 0 && hSplash > 0) {
+                if (option.find("-splash:") == 0 && hSplash != (void *) NULL) {
```

Comment:
Can this use `nullptr` instead of `NULL`, or is the used **C++** version too old for that?

```suggestion
                if (option.find("-splash:") == 0 && hSplash != nullptr) {
```

Reply:
> or is the used C++ version too old for that?

@ExE-Boss hi! The build image is locked to [ubuntu 22](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#language-and-runtime) since the last LTS is soon eol. So the tooling env isn't super old.

I can change this if this is the cleaner way to do that comparison. Thanks for the review btw!